### PR TITLE
Fix Instagram

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

File: declarations/Instagram.filters.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export function removeUnessecaryHrefHQueryParam(document) {
+  document.querySelectorAll("a").forEach((el) => {
+    const href = el.getAttribute("href");
+    const params = new URLSearchParams(href);
+    if (params.has("h") === true) {
```
Comment (martinratinaud) [inline declarations/Instagram.filters.js:5]: no need for `===true` I guess

File: declarations/Instagram.filters.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+      params.delete("h");
+      el.setAttribute("href", params.toString());
```
Comment (martinratinaud) [inline declarations/Instagram.filters.js:7]: here you seem to replace href with only the search params

Best way would maybe to use `new URL(href)`

**Review by martinratinaud — CHANGES_REQUESTED** [review]
I think it does not work the way you did. :-)

File: declarations/Instagram.filters.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export function removeUnessecaryHrefHQueryParam(document) {
```
Comment (MattiSG) [inline declarations/Instagram.filters.js:1]:
```suggestion
export function removeTrackingIDs(document) {
```

File: declarations/Instagram.json [hunk 4]
```diff
@@ -3,13 +3,14 @@
   "documents": {
     "Privacy Policy": {
       "fetch": "https://help.instagram.com/519522125107875/?locale=fr_FR",
-      "select": [".d5zfiv1f"],
+      "select": ["div[role=\"main\"]"],
```
Comment (MattiSG) [inline declarations/Instagram.json:6]: > Do we really need to quote the attribute value?
```suggestion
      "select": ["div[role=main]"],
```
> I'm really not sure about that with pupetteer, please check it if you're interested in the change!

**Review by MattiSG — COMMENTED** [review]
Can't wait for Instagram to stop blinking 😊

Congratulations for finding a more stable identifier than the React-generated classes!

File: declarations/Instagram.json (hunk 4 above)
Comment (clementbiron) [reply declarations/Instagram.json]: I just tested it and it works. It's much clearer thanks.

**Pull request merged**